L/LRSLib_PolyhedraFork/bundled/makefile.liblrsnash

```diff
@@ -0,0 +1,7 @@
+INCLUDEDIR = /usr/include
```

Comment:
This is wrong, we shouldn't use system directories.  Also, if you use the [environment variables](https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables) already defined in the environment (`includedir`, `libdir`, etc) you don't have to define anything here